client/src/pages/Home.js

```diff
@@ -2,6 +2,9 @@ import { Link } from "react-router-dom";
 import { MenuBar } from "../components/MenuBar";
 import "./Home.css";
 import { PicCarousel } from "../components/PicCarousel";
+import { VideoWidget } from "../components/VideoWidget";
+import { Footer } from "../components/Footer";
+import "./VideoWidget.css";
```

Comment:
You can also move this css file to the component directory, the idea is that you would import the VideoWidget.css in the VideoWidget.js just like they did in the Home.js where the only css import was Home.css